api/src/main/java/org/ehcache/config/Eviction.java

```diff
@@ -35,6 +35,7 @@ public boolean adviseAgainstEviction(Object key, Object value) {
    * @param <V> the value type for the advisor
    * @return an advisor where no mappings are advised against eviction
    */
+  @SuppressWarnings("unchecked")
```

Comment:
No.  It needs to be typed. However, typing K, V will allow noAdvice to be any wanted type